addons/udes_stock/views/stock_picking.xml

```diff
@@ -158,6 +158,30 @@
                 <attribute name="attrs">{'readonly': [('state', '!=', 'draft')],  'required': [('picking_type_code', '=', 'incoming')]}</attribute>
             </xpath>
 
+            <xpath expr="//field[@name='date_done']" position="attributes">
+                <attribute name="attrs">{'invisible': [('state', 'not in', ('done', 'cancel'))]}</attribute>
+                <attribute name="string">Date of Transfer</attribute>
```

Comment:
It is annoying to have to explicity set the string back to what it is set in the Python code.

Could you try the following and see if it works?

`<attribute name="string"/>`

Reply:
In fact on field definition on python it was Date of Transfer, but on xml in the form view was changed to Effective Date.
We discussed in the morning and i remember that you said better on python to be same on extended filters

Reply:
Yeah, so the name on the Python side is fine. What I mean is that here we are now manually setting it in the XML even though it is the same as in Python. So that is why I'm wondering if we could just remove the `string` attribute on the XML view.